src/react-ui-components/src/structure/sideToolbar.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import { observer } from 'mobx-react';
+import { withStyles } from '@material-ui/core/styles';
+import classNames from 'classnames';
+import Drawer from '@material-ui/core/Drawer';
+import Divider from '@material-ui/core/Divider';
+import ToolBarButton from 'react-ui-components/buttons/toolbarButton';
+
+const styles = theme => {
+    const drawerWidth = 240;
+    const margin = theme.direction === 'rtl' ? 'marginLeft' : 'marginRight';
+    return {
+        drawerClass: {
+            height: '100%',
+            float: 'right'
+        },
+        drawerPaper: {
+            position: 'relative',
+            whiteSpace: 'nowrap',
+            width: drawerWidth,
+            transition: theme.transitions.create('width', {
+                easing: theme.transitions.easing.sharp,
+                duration: theme.transitions.duration.enteringScreen,
+            }),
+        },
+        drawerPaperClose: {
+            overflowX: 'hidden',
+            transition: theme.transitions.create('width', {
+                easing: theme.transitions.easing.sharp,
+                duration: theme.transitions.duration.leavingScreen,
+            }),
+            width: theme.spacing.unit * 7,
+            [theme.breakpoints.up('sm')]: {
+                width: theme.spacing.unit * 9,
+            },
+        },
+        content: {
+            flexGrow: 1,
+            backgroundColor: theme.palette.background.default,
+            padding: theme.spacing.unit * 3
+        },
+        ic_validateForm: {
```

Comment:
is this class name? (can be use in css selector),
if yes, should be as css-naming-conventions (ic-validate-form)